### PR TITLE
initial frontend plugin to backend

### PR DESCRIPTION
**Pull request by okunator**
plus some minor refactoring

### CODE REVIEW

File: oncodash-app/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
   </style>
   <title>oncodash-app</title>
 </head>
-
+<h1>Hello Oncodash</h1>
```
Comment (tuner) [inline oncodash-app/index.html:21]: This should be inside the `<body>` tag :)

Comment (okunator) [reply oncodash-app/index.html]: Resolved here: https://github.com/oncodash/oncodash/pull/15/commits/90e125533519d6505a30500f7683b93f25a15205 :)

**Comment by tuner** [comment]
You could fix the indentation.

1. Install https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig ... which reads the indentation settings, etc. from `.editorconfig`
2. Install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode ... and configure it to format on save

You can also run `npm run format` which runs prettier and fixes indentation and other formatting issues.

**Comment by jdreo** [comment]
Merged.

**Pull request closed without merging**